### PR TITLE
 Add Support for Surface Pro 9 Intel, Surface Laptop 5

### PR DESCRIPTION
**Pull request by zayedrasha**
Description:
- Add Surface Pro 9 Intel and Surface laptop 5 to the supported devices on CreateSurfaceWindowsImage.ps1.
- Add Drivers list for each device.
- Update README.md

### CODE REVIEW

File: CreateSurfaceWindowsImage.ps1 [hunk 1]
```diff
@@ -179,7 +183,7 @@ Param(
         Mandatory=$False,
         HelpMessage="Surface device type to add drivers to image for, if not specified no drivers injected - Custom can be used if using with a non-Surface device"
         )]
-        [ValidateSet('SurfacePro4', 'SurfacePro5', 'SurfacePro6', 'SurfacePro7', 'SurfacePro7Plus', 'SurfacePro8', 'SurfaceLaptop', 'SurfaceLaptop2', 'SurfaceLaptop3Intel', 'SurfaceLaptop3AMD', 'SurfaceLaptop4Intel', 'SurfaceLaptop4AMD', 'SurfaceLaptopGo', 'SurfaceLaptopStudio', 'SurfaceBook', 'SurfaceBook2', 'SurfaceBook3', 'SurfaceStudio', 'SurfaceStudio2', 'SurfaceGo', 'SurfaceGoLTE', 'SurfaceGo2', 'SurfaceGo3', 'SurfaceHub2', 'Custom')]
+        [ValidateSet('SurfacePro4', 'SurfacePro5', 'SurfacePro6', 'SurfacePro7', 'SurfacePro7Plus', 'SurfacePro8', 'SurfacePro9', 'SurfaceLaptop', 'SurfaceLaptop2', 'SurfaceLaptop3Intel', 'SurfaceLaptop3AMD', 'SurfaceLaptop4Intel', 'SurfaceLaptop4AMD', 'SurfaceLaptop5', 'SurfaceLaptopGo', 'SurfaceLaptopStudio', 'SurfaceBook', 'SurfaceBook2', 'SurfaceBook3', 'SurfaceStudio', 'SurfaceStudio2', 'SurfaceGo', 'SurfaceGoLTE', 'SurfaceGo2', 'SurfaceGo3', 'SurfaceHub2', 'Custom')]
```
Comment (lnaugle) [inline CreateSurfaceWindowsImage.ps1:186]: SurfacePro9 should be updated to SurfacePro9Intel

Comment (lnaugle) [reply CreateSurfaceWindowsImage.ps1:186]: Also around line 1425 please add a special case to properly add intel to the MSI aka.ms link.

Comment (zayedrasha) [reply CreateSurfaceWindowsImage.ps1]: Updated

Comment (paulfromholland) [reply CreateSurfaceWindowsImage.ps1]: Does SurfaceLaptop5 now works?

**Pull request closed without merging**